examples/OECC/config/rc.local

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
```

Comment:
Is this intentionally /bin/sh or do we need to standardize on /bin/bash given past confusion on Ubuntu?

Reply:
It is intentional.
The idea is that on OS startup, /bin/sh (/bin/dash) is a much lighter weight executable than /bin/bash.